delta/data/preprocess/text_ops.py

```diff
@@ -17,17 +17,21 @@
 
 import tensorflow as tf
 from absl import logging
+import os
 
 from delta import utils
 from delta.layers.ops import py_x_ops
+from delta.data.utils import read_lines_from_text_file
```

Comment:
make `read_lines_from_text_file` a local func.

Reply:
It was also used in `jieba_op_test.py`.